src/pages/groupComparison/ComparisonGroupManager.tsx

```diff
@@ -0,0 +1,285 @@
+                                onClick={()=>{
+                                    this.recentlyDeleted = false;
+                                    this.props.store.toggleComparisonGroupSelected(group.id);
+                                }}
+                            />{group.name} ({group.sampleIdentifiers.length})
+                        </label></div>
+                    ))
+                )}
+            </div>
+        );
+    }
+    private get selectButton() {
+        return (
+            <button
+                className="btn btn-sm btn-primary"
+                onClick={()=>{
+                    this.props.store.clearAllFilters();
+                    const ids:SampleIdentifier[] =
+                        _.uniqWith(
+                            _.flattenDeep<SampleIdentifier>(this.props.store.selectedComparisonGroups.map(group=>group.sampleIdentifiers)),
+                            (id1, id2)=>((id1.sampleId === id2.sampleId) && (id1.studyId === id2.studyId))
+                        );
+                    this.props.store.updateChartSampleIdentifierFilter(UniqueKey.SELECTED_COMPARISON_GROUPS, ids);
+                }}
+            >Select</button>
+        );
+    }
+    private get compareButton() {
+        return (
+            <button className="btn btn-sm btn-primary"
+                    disabled={this.props.store.selectedComparisonGroups.length < 2}
+                    onClick={()=>{
+                        window.open(getComparisonUrl({localGroups:this.props.store.selectedComparisonGroups.map(group=>group.id).join(",")}), "_blank");
+                    }}
+            >Compare</button>
+        );
+    }
+    private get restoreOrDeleteButton() {
+        if (this.recentlyDeleted) {
+            return (
+                <button
+                    className="btn btn-sm btn-primary"
+                    onClick={()=>{
+                        restoreRecentlyDeletedGroups();
+                        this.recentlyDeleted = false;
+                    }}
+                >Restore</button>
+            );
+        } else {
+            return (
+                <button
+                    className="btn btn-sm btn-primary"
+                    onClick={()=>{
+                        const groupIds = this.props.store.selectedComparisonGroups.map(group=>group.id);
+                        for (const groupId of groupIds) {
+                            this.props.store.removeComparisonGroupSelectionEntry(groupId);
+                        }
+                        deleteGroups(groupIds);
+                        this.recentlyDeleted = true;
+                    }}
+                    disabled={this.props.store.selectedComparisonGroups.length === 0}
+                >Delete</button>
+            );
+        }
+    }
+
+    private get actionButtons() {
+        return (
+            <div
+                style={{
+                    position:"absolute",
```

Comment:
lets discuss why position:absolute.    It should generally be avoided except when you need to position something outside the "flow" of the DOM.